app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @project_material = @project.material
+    @project_challenge_theme = @project.challenge.theme
+    @contestant_count = @project.contestant_count
+    @average_years_exp = @project.average_years_exp
```

Comment:
Great job making these methods in the Project model. I would also make a method in the Project model called `challenge_theme` that would then grab this data. Law of Demeter dictates that we should ask each object that owns the attribute about that attribute as opposed to chaining through objects to dig for that info. Very slight difference and if that's not making sense, just ask me! :)